examples/mobile/HomeApp/js/app.js

```diff
@@ -30,7 +30,7 @@ import Storage from "./clipping-storage.js";
 			]
 		},
 		{
-			"appID": "vUf39tzQ4s.UIComponents",
+			"appID": "vUf39tzQ4s.Netflix",
```

Comment:
are you changing the base to netflix?

Reply:
This app ID in this file has no real association with applications. It just has to be different.